src/modules/uORB/uORBDeviceNode.hpp

```diff
@@ -195,6 +195,11 @@ class uORB::DeviceNode : public cdev::CDev, public ListNode<uORB::DeviceNode *>
 	int get_priority() const { return _priority; }
 	void set_priority(uint8_t priority) { _priority = priority; }
 
+	hrt_abstime last_update();
+
+	bool copy(void *dst, unsigned &generation);
```

Comment:
> * The class API docs don't make it clear when to use which class. Can you clarify this? Especially that `Subscription` should generally be used over the Polled classes.
> * Can you rename `SubscriptionPolled` to `SubscriptionPollable`? I find `Polled` confusing, since there is no `poll` involved.

The naming is better, but I'd actually like to kill this off entirely as soon as possible (likely once FlightTasks is migrated). Then it's only LPE using it for polling, which we can either work around with the C API, or add a blocking update method to the new uORB::Subscription. 

> * Did you check the need for `virtual` for the `Subscription` classes? If not required, we should remove that overhead early on.

`uORB::SubscriptionInterval` overrides update, but let's revisit after logger has been migrated and see what we really need. I'd still like to get to a template uORB::Subscription (without the internal data).

Reply:
Added.